frontend/packages/console-shared/src/components/dashboard/dashboard-card/card.scss

```diff
@@ -6,6 +6,19 @@ $co-dashboard-card--line-height: 19px;
   margin: 0;
 }
 
+.co-dashboard-card--gradient {
+  position: relative;
+  &:after {
+    content: '';
+    position: absolute;
```

Comment:
```suggestion
    pointer-events: none;
    position: absolute;
```

Based on [Matt's CodePen ](https://codepen.io/matthewcarleton/pen/jOEWzJd?editors=1100) it does seem like we need `pointer-events: none;` here. Should these properties be alphabetized by the way?

Reply:
we could also add `width: calc(100% - var(--pf-global--spacer--md));` here which really doesn't look that bad.
<img width="505" alt="Screen Shot 2019-12-10 at 10 59 14 AM" src="https://user-images.githubusercontent.com/287880/70540725-23c01500-1b3c-11ea-9784-f6a7735ea118.png">

Reply:
`pointer-events: none;` added